composer.json

```diff
@@ -33,10 +33,10 @@
         "mockery/mockery": "0.9.*",
         "monolog/monolog": "*",
         "sensio/framework-extra-bundle": "^3.0",
-        "symfony/symfony": "^2.8||^3.0",
-        "symfony/phpunit-bridge": "^3.2",
-        "symfony/expression-language": "^2.4||^3.0",
-        "symfony/monolog-bundle": "^2.3||^3.0",
+        "symfony/symfony": "^2.8||^3.0||^4.0",
```

Comment:
It should not require symfony/symfony. That defeats the whole purpose of requiring the individual components. It's just require-dev but still wrong.

Reply:
indeed. we should use something like dunglas/symfony-lock in the ci instead, so that we see if our require deps are all that is needed.